migrations/migration_0001_move_selected_collections.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
This migration doesn't use any sort of migrations framework, it's just a standlone script. This seemed simplest, as I don't think any of our existing dependencies provide a framework for this.